gdx/src/com/badlogic/gdx/utils/ScreenUtils.java

```diff
@@ -27,12 +28,41 @@
 import com.badlogic.gdx.graphics.g2d.TextureRegion;
 import com.badlogic.gdx.math.MathUtils;
 
-/** Class with static helper methods that provide access to the default OpenGL FrameBuffer. These methods can be used to get the
+/** Class with static helper methods related to OpenGL buffers, including access to the default OpenGL FrameBuffer. These methods can be used to get the
  * entire screen content or a portion thereof.
  * 
  * @author espitz */
 public final class ScreenUtils {
 
+	/** Clears the color buffer with the specified color. */
+	public static void clearColor (float r, float g, float b, float a) {
+		Gdx.gl.glClearColor(r, g, b, a);
+		Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
+	}
+
+	/** Clears the color buffer with the specified color. */
+	public static void clearColor (Color color) {
+		clearColor(color.r, color.g, color.b, color.a);
+	}
+
+	/** Clears the current render buffer. 
+	 * @param colorClear Clears the color buffer if true.
+	 * @param depthClear Clears the depth buffer if true.
+	 * @param color The color to clear with, used only if clearColor is true.
+	 * @param depth The depth to clear the z-buffer with, used only if clearDepth is true. The valid range is from 0 to 1.
+	 * */
+	public static void clear (boolean colorClear, boolean depthClear, Color color, float depth) {
+		int mask = 0;
+		if (colorClear)
+			mask = mask | GL20.GL_COLOR_BUFFER_BIT;
+			Gdx.gl.glClearColor(color.r, color.g, color.b, color.a);

```

Comment:
Indentation like this is not good, makes it look like the if has curlies, but it doesn't.

Reply:
Yep, curlies were missing, removed them at some point and forgot to add them back when I added the second line (I'm not used of not adding them). Thanks!